modules/custom/d_p/d_p.module

```diff
@@ -145,9 +158,14 @@ function d_p_form_alter(&$form, FormStateInterface $form_state, $form_id) {
 
   // Add bootstrap classes to CTA by default.
   $trigger_name = $form_state->getUserInput()['_triggering_element_name'] ?? NULL;
-  if (strpos($form_id, 'add') !== FALSE || strpos($trigger_name, 'add') !== FALSE) {
+  if (
+    strpos($form_id, 'add') !== FALSE ||
+    strpos($trigger_name, 'add') !== FALSE ||
+    strpos($form_id, 'edit') !== FALSE ||
+    strpos($trigger_name, 'edit') !== FALSE
+  ) {
     if (isset($form['field_d_cta_link'])) {
-      $form['field_d_cta_link']['widget'][0]['options']['attributes']['class']['#default_value'] = 'btn btn-primary';
+      $form['field_d_cta_link']['widget'][0]['options']['attributes']['class']['#default_value'] = add_cta_link_default_class($form['field_d_cta_link']['widget'][0]['options']['attributes']['class']['#default_value']);
     }
     edit_cta_link_default_class($form);
```

Comment:
tak, to jest do modycikacji, w przypadku edycji, trzeba sprawdzic, czy cos jest juz ustawione pole CTA ( czy ma wartość) jak ma jakiś link, to nic już nie robimy - user mógł zmodyfikować te klasy i nie zmieniamy nic.